pkg/feature/raw_resources.go

```diff
@@ -42,18 +42,25 @@ func createResources(cli client.Client, objects []*unstructured.Unstructured, me
 		namespace := object.GetNamespace()
 
 		err := cli.Get(context.TODO(), k8stypes.NamespacedName{Name: name, Namespace: namespace}, object.DeepCopy())
-		if err == nil {
-			// object already exists, skip reconcile allowing users to tweak it
-			continue
-		}
-		if !k8serrors.IsNotFound(err) {
-			return errors.WithStack(err)
+		if err != nil {
+			if k8serrors.IsNotFound(err) {
+				// object does not exist and should be created
+				if err = cli.Create(context.TODO(), object); err != nil {
+					return fmt.Errorf("failed to create object %s/%s: %w", namespace, name, err)
+				}
+				continue
+			}
+			return fmt.Errorf("failed to get object %s/%s: %w", namespace, name, err)
```

Comment:
It's really hard to review on mobile but here's what I think we can do 

```golang
err := cli.Get(...);

if client.IgnoreNotFound(err) != nil {
   return err

}

if err != nil { // safe to assume not found, we covered all the other cases above
   Create and fail if client.IgnoreAlreadyExists(err) != nil
}

If managed - Update/Patch

```

I'm extremely tired so take with the grain of salt. But wanted to clarify my review from the morning.

I hope @aslakknutsen can help validating my hallucinations.